crates/collab_ui/src/collab_titlebar_item.rs

```diff
@@ -59,26 +58,14 @@ impl Render for CollabTitlebarItem {
         let project_id = self.project.read(cx).remote_id();
         let workspace = self.workspace.upgrade();
 
-        h_flex()
-            .id("titlebar")
-            .justify_between()
-            .w_full()
-            .h(titlebar_height(cx))
-            .map(|this| {
-                if matches!(cx.window_bounds(), WindowBounds::Fullscreen) {
-                    this.pl_2()
-                } else {
-                    // Use pixels here instead of a rem-based size because the macOS traffic
-                    // lights are a static size, and don't scale with the rest of the UI.
-                    this.pl(px(80.))
-                }
-            })
-            .bg(cx.theme().colors().title_bar_background)
-            .on_click(|event, cx| {
-                if event.up.click_count == 2 {
-                    cx.zoom_window();
-                }
-            })
+        platform_titlebar()
+            .titlebar_bg(cx.theme().colors().title_bar_background)
+            // .on_click(|event, cx| {
+            //     if event.up.click_count == 2 {
+            //         cx.zoom_window();
+            //     }
+            // })
```

Comment:
this is already handled on windows. any advice on how to handle this for mac/linux would be appreciated. I could try to add the `Clickable` trait to the platform titlebar?

Reply:
This should be incorporated into `platform_titlebar()`, right?